oak_functions/examples/key_value_lookup/example.toml

```diff
@@ -0,0 +1,18 @@
+name = "key_value_lookup"
+
+[applications]
+
+[applications.rust]
+type = "Functions"
+wasm_path = "oak_functions/examples/bin/key_value_lookup.wasm"
+target = { Cargo = { cargo_manifest = "oak_functions/examples/key_value_lookup/module/Cargo.toml" } }
+
+[server]
+additional_args = [
+  "--config-path=./oak_functions/examples/key_value_lookup/config.toml"
+]
+
+[clients]
+rust = { Cargo = { cargo_manifest = "oak_functions/client/Cargo.toml" }, additional_args = [
```

Comment:
I assume this sends a request with an empty body. It might be useful for testing to add support for piping a value into the client via stdin in future.